README.md

```diff
@@ -113,11 +88,7 @@ To **test** if your file gets processed properly you can do the following steps:
 </p>
 
 ### PDF
-<p align="center">
-  <img width="100%" src="doc/diagramms/pdf.svg" alt="PDF diagramm">
-</p>
-
-**Note on PDF processing:** since the processing algorithm for PDF files makes heavy use of splitting an recombining the single PDF pages, it could damage certain PDF files or manipulate the content somehow. 
+For processing PDF files the external commandline tool [`OCRmyPDF`](https://github.com/jbarlow83/OCRmyPDF) is used. The tool is invoked with the [`--skip-text`](https://ocrmypdf.readthedocs.io/en/latest/advanced.html#when-ocr-is-skipped) parameter so that it will skip pages which already contain text. Therefore it is possible to process PDF files containing both "born digital" and scanned content.
```

Comment:
This paragraph is not correct since the `--skip-text` parameter submitted to `OCRmyPDF` ignores images embedded into a text page. For this purpose, use the `--redo-ocr` parameter instead.

```suggestion
For processing PDF files, the external command line tool [`OCRmyPDF`](https://github.com/jbarlow83/OCRmyPDF) is used. The tool is invoked with the [`--redo-ocr`](https://ocrmypdf.readthedocs.io/en/latest/advanced.html#when-ocr-is-skipped) parameter so that it will perform a detailed text analysis. The detailed analysis masks out visible text and sends the image of each page to the OCR processor. After processing, additional text is inserted as OCR, whereas existing text in a mixed file document (images embedded into text pages) is not disrupted.
```